CHANGELOG.md

```diff
@@ -604,7 +607,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Removed `TrainerProperties` mixin and moved property definitions directly into `Trainer` ([#9495](https://github.com/PyTorchLightning/pytorch-lightning/pull/9495))
 - Removed a redundant warning with `ModelCheckpoint(monitor=None)` callback ([#9875](https://github.com/PyTorchLightning/pytorch-lightning/pull/9875))
 - Remove `epoch` from `trainer.logged_metrics` ([#9904](https://github.com/PyTorchLightning/pytorch-lightning/pull/9904))
-- Removed `should_rank_save_checkpoint` property from Trainer ([#9433](https://github.com/PyTorchLightning/pytorch-lightning/pull/9433))
```

Comment:
Oh, I didn't notice this had been already included in the 1.5 release.
In that case, it's too late to deprecate to avoid breaking BC.

@ananthsub do you still think we should add this back? If yes, then this should be included in a bug-fix release

Reply:
Oh good catch @carmocca - I didn't realize it was removed already in the public release either. In that case, I don't think we should add it back as the breaking change is already done :/

Reply:
changelog still has this "removal" listed but it wasn't actually removed bc of the merging mistake? (described in #11035)